### PR TITLE
[032] Update readme to include commands to run test scripts

### PR DESCRIPTION
**Pull request by TaylorJCrawford**

### CODE REVIEW

**Review by siyandlovu1 — COMMENTED** [review]
Able to run the test following the steps in the readme

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+This application provides a frontend for the REST API: https://github.com/TaylorJCrawford/the-it-crowd-api
+
+Follow the steps in the above link to run the REST API
+
+
+How to start the the-it-crowd-ui application
+---
+
+1. Run `npm install` to build your application
+2. Start application with API Repo (See ReadMe In Repo)
```
Comment (glenn-hamilton) [inline README.md]: This doesn't tell us how to start the project

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+Run Unit Tests
+---
+
+To run unit tests run the following command:
+
+```shell
+npm test
+```
+
+Run Unit Tests
```
Comment (glenn-hamilton) [inline README.md]: This should be UI tests

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
...
+---
+
+To run UI tests locally, change the URL to run against remote following command:
+
+```shell
+UI_TEST_URL=http://localhost:3000 && npm run test-ui
```
Comment (glenn-hamilton) [inline README.md]: This would be better being split up into an environment variable section

Comment (TaylorJCrawford) [reply README.md]: Going to leave out until UI Test Are Added. Haven't decided on ENV name, etc...

File: README.md (hunk 2 above)
Comment (TaylorJCrawford) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (TaylorJCrawford) [reply README.md]: Fixed

**Review by siyandlovu1 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,34 @@
...
+npm test
+```
+
+Run UI Tests
+---
+
+To run UI tests locally, run the following command:
+
+```shell
+npm run test-ui
```
Comment (glenn-hamilton) [inline README.md:49]: This command currently won't do anything?

File: README.md [hunk 5]
```diff
@@ -1 +1,34 @@
...
+
+Follow the steps in the above link to run the REST API
+
+
+How to start the the-it-crowd-ui application
+---
+
+1. Run `npm install` to build your application
+2. Start application with `npm run dev`
+3. To check that your application is running enter url `http://localhost:3000`
```
Comment (glenn-hamilton) [inline README.md:32]: You've mentioned the localhost URL, could you also link to the AWS URL as well

File: README.md [hunk 6]
```diff
@@ -1 +1,34 @@
-# the-it-crowd-ui
\ No newline at end of file
+# the-it-crowd-ui
```
Comment (glenn-hamilton) [inline README.md:1]: Are there any environment variables that are needed to run the frontend?

File: README.md (hunk 5 above)
Comment (TaylorJCrawford) [reply README.md:32]: Afraid not. Going to add this with the Docker part to keep in line with the scope of the ticket.

File: README.md (hunk 4 above)
Comment (TaylorJCrawford) [reply README.md:49]: Correct. Ticket has not been blocked until US001 is in main. Then this command will be able to be run.

**Review by glenn-hamilton — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -1,6 +1,54 @@
...
+2. Start application with `npm run dev`
+3. To check that your application is running enter url `http://localhost:3000`
+
+
+# Run Unit Tests
+
+To run unit tests run the following command:
+
+```shell
+npm test
```
Comment (siyandlovu1) [inline README.md]: This Test would run for me however "npm run test-unit" worked for me

File: README.md [hunk 8]
```diff
@@ -1,6 +1,54 @@
...
+
+This application provides a frontend for the REST API: https://github.com/TaylorJCrawford/the-it-crowd-api
+
+Follow the steps in the above link to run the REST API
+
+
+# How to start the the-it-crowd-ui application
+
+1. Run `npm install` to build your application
+2. Start application with `npm run dev`
```
Comment (siyandlovu1) [inline README.md]: This Didnt start the app on my system however "npm start" did

**Pull request merged**